firebase-installations/src/test/java/com/google/firebase/installations/FirebaseInstallationsTest.java

```diff
@@ -639,13 +554,13 @@ public void testGetAuthToken_expiredAuthToken_fetchedNewTokenFromFIS() throws Ex
         PersistedInstallationEntry.INSTANCE.withRegisteredFid(
             TEST_FID_1,
             TEST_REFRESH_TOKEN,
-            utils.currentTimeInSecs(),
+            // Make experation time to be 30 min ago
+            utils.currentTimeInSecs()
+                - TEST_TOKEN_EXPIRATION_TIMESTAMP
+                - TimeUnit.MINUTES.toSeconds(30),
```

Comment:
make this + TimeUnit.MINUTES.toSeconds(30) please
FIS SDK should update a FIS auth token even if it is still valid for another hour...
(see grace period)

Reply:
done